Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -374,13 +397,18 @@ var Process = (function () {
             $(profilingButton).addClass("btn-danger");
         }
 
+        var iisProfilingCheckbox = Utilities.getCheckbox(this._json.id + "-iisProfilingCheck", "Collect IIS Events")
+       
         if (Process.prototype.WebSiteSku === "Free" || Process.prototype.WebSiteSku === "Shared") {
```

Comment:
The profiling checkbox should also be disabled(not clickable) for free/shared. Similar to what we are doing for the button.